dowhy/utils/graph_operations.py

```diff
@@ -1,6 +1,7 @@
 import re
 from queue import LifoQueue
 
+import graphviz
```

Comment:
This import should stay within the method, since `graphviz` is an optional package, which would cause a module error for users who didn't install it.